packages/migrate/src/__tests__/DbSeed.test.ts

```diff
@@ -18,6 +18,18 @@ describe('seed', () => {
     expect(ctx.mocked['console.error'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
   })
 
+  it('seed.js with extra args', async () => {
+    ctx.fixture('seed-sqlite-js-extra-args')
+
+    const result = DbSeed.new().parse(['--', '--my-custom-arg-from-cli=my-value', '-z'])
+    await expect(result).resolves.toContain(`The seed command has been executed.`)
+    expect(ctx.mocked['console.info'].mock.calls.join('\n')).toMatchInlineSnapshot(
+      `Running seed command \`node prisma/seed.js --my-custom-arg-from-config=my-value --my-custom-arg-from-cli=my-value -z\` ...`,
+    )
+    expect(ctx.mocked['console.log'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
+    expect(ctx.mocked['console.error'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
+  })
```

Comment:
This is a test for the successful case, we also need a test for the error case (as per [Slack](https://prisma-company.slack.com/archives/C02FNFLDUS3/p1685006712466749?thread_ts=1685004806.833549&cid=C02FNFLDUS3))

Reply:
What if args are passed as `--my-custom-arg-from-cli my-value -z`, i.e., without the `=` symbol?